Create kochava-event-streaming-post-installs.md (#932)
Amplitude-Developer-Docs/amplitude-dev-center: docs/data/destinations/kochava-event-streaming-post-installs.md

    Within this documentation, "Kochava (Post-Installs)" focuses on events unfolding after app installations. Here, the "Send User" feature comes into play, establishing essential links between identities and actions. On the other hand, "Kochava (Installs) events streaming" centers around initial app installations. This segment lacks the "Send User" feature, aligning with the prerequisite of timely install data for successful IdentityLink integration.

## Considerations

- You must enable this integration in each Amplitude project you want to use it in.
- You need a Kochava account to enable this integration.

> - Amplitude has two destinations for **Kochava (Install)** and Kochava **(Post Install)**.

Reviewer: ```suggestion
- Amplitude has two destinations for **Kochava (Install)** and **Kochava (Post Install)**.
```

  - The requests must be smaller than 2MB.

## Setup

### Prerequisites

To configure an Event Streaming integration from Amplitude to Kochava, you must fulfill the following prerequisites from Kochava:

- **A Kochava Account:** You must have a Kochava account to leverage this integration. Contact Kochava for more details.
- **Kochava App GUID:** You must create an App and have the GUID assigned to the App.
- **(Optional) Kochava API Key:** If you want to enable the authentication with the integration, you have to add your API Key in the Account Option.
- **(Optional) Kochava secret authentication key:** If you want to enable the authentication with the integration, you have to request a secret authentication key through the Kochava Client Success Team.

### Kochava setup

1. Log into your Kochava dashboard and copy your **Account ID** and **Account Passcode**.
2. Go to the All **Apps** section and copy the GUID of the App you created.
3. (Optional) Go to **Account Options** and select **API Keys** to locate or create your API key.
4. (Optional) Contact your Kochava Client Success Manager to obtain your Kochava secret authentication key.

### Amplitude setup

> 1. In Amplitude, navigate to **Data Destinations**, then find **Kochava (Install)**.

Reviewer: ```suggestion
1. In Amplitude, navigate to **Data Destinations**, then find **Kochava (Post Install)**.
```

2. Enter a sync name, then click **Create Sync**.
3. Toggle Status from **Disabled** to **Enabled**.
4. Paste your **App GUID**.
5. *(Optional)* Click the single box **Enable Authentication** if you want to authenticate your sync.
6. *(Optional)* Paste your **API Key** (taken from the Kochava account).
7. *(Optional)* Paste your secret key to **API secret** (provided by Kochava's CS team).
8. Under **Send Events**, make sure the toggle is enabled ("Events are sent to Kochava") if you want to stream events to Kochava. When enabled, events are automatically forwarded to Kochava when they're ingested in Amplitude. Events aren't sent on a schedule or on demand using this integration.
9. In **Select and Filter** events choose which events you want to send. Choose only the events you need in Kochava. [Transformed events](https://help.amplitude.com/hc/en-us/articles/5913315221915-Transformations-Retroactively-modify-your-event-data-structure#:~:text=Amplitude%20Data's%20transformations%20feature%20allows,them%20to%20all%20historical%20data.) aren't supported.

> 10. Click on **Map properties to destination** to specify the identity mapping between Amplitude and Kochava. 

Reviewer: ```suggestion
10. Click on **Mappings** to specify the identity mapping between Amplitude and Kochava.  You must choose at least one of the following identifiers, Apple Advertising ID (idfa), Apple Vendor ID (idfv), Google Advertising ID (adid) and Android ID (android_id). 
```
